package.json

```diff
@@ -9,7 +9,7 @@
     "integromat",
     "imt"
   ],
-  "version": "2.4.0",
+  "version": "2.5.0",
```

Comment:
Unless you see something here that would require a Major version bump. To me seems like minor